integration/utils/config.go

```diff
@@ -17,10 +17,15 @@ package utils
 import (
 	"fmt"
 	"path/filepath"
+	"strings"
 )
 
 // Config represents the configuration for the integration tests as provided in the command line.
 type Config struct {
+	// features contains the set of features enabled during the build.  This represents a set
+	// and therefore the values in the map are meaningless.
+	features map[string]bool
```

Comment:
Why unexported?

Reply:
Because I added the `HasFeature` method below to query this.

Reply:
Meh. I don't think you get anything out of using an accessor here, but up to you.

Reply:
Yeah, fair enough. Removed.